pkgs/build-support/cc-wrapper/macos-sierra-reexport-hack.bash

```diff
@@ -0,0 +1,103 @@
+#! @shell@
+
+set -eu -o pipefail
+
+path_backup="$PATH"
+if [ -n "@coreutils_bin@" ]; then
+  PATH="@coreutils_bin@/bin"
+fi
+
+declare -r recurThreshold=300
+
+declare overflowCount=0
+for ((n=0; n < $#; ++n)); do
+    case "${!n}" in
+        -l*) let overflowCount+=1 ;;
+        -reexport-l*) let overflowCount+=1 ;;
+        *) ;;
```

Comment:
You mean just plain paths to dylibs passed in? I didn't see any `--` in the man page.